### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@yuliu-okta think of this as a rebase. Shipit, or are there still issues?  Thanks!

### CODE REVIEW

File: _source/_docs/api/resources/resource-server-beta/required-config-changes-for-auth-server.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ title: Configuration Changes Required for API Access Management Beta Update
...
 ## Configuration Changes Required for API Access Management Beta Update
 
-Okta provides the private authorization server for OAuth 2.0 (Beta).
+Okta provides a private authorization server for OAuth 2.0 (Beta).
 However, with release 2016.45, you can use a shared authorization server instead, which provides several benefits:
 
 * You created a separate private authorization server for each client, which required you to set scopes and claims 
   and group filters for each client on a separate server, but keeping many configurations in sync is challenging.
-  With the shared resource server, you can create a single configuration of scopes, claims, and Group filters, and
+  With the shared authorization server, you can create a single configuration of scopes, claims, and Group filters, and
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/resource-server-beta/required-config-changes-for-auth-server.md]: why Group is capital?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/resource-server-beta/required-config-changes-for-auth-server.md]: Will fix group capitalization--and sorry, I thought I addressed all your issues! Will go back and check.

Comment (yuliu-okta) [inline _source/_docs/api/resources/resource-server-beta/required-config-changes-for-auth-server.md]: @mystiberry-okta here is other comments:
1. https://github.com/okta/okta.github.io/pull/451#discussion_r86613820
2. https://github.com/okta/okta.github.io/pull/451#discussion_r86617318
3. https://github.com/okta/okta.github.io/pull/451#discussion_r86617501
4. https://github.com/okta/okta.github.io/pull/451#discussion_r86617979
5. https://github.com/okta/okta.github.io/pull/451#discussion_r86619376
6. https://github.com/okta/okta.github.io/pull/451#discussion_r86619508
7. https://github.com/okta/okta.github.io/pull/451#discussion_r86619600
8. https://github.com/okta/okta.github.io/pull/451#discussion_r86619766
9. https://github.com/okta/okta.github.io/pull/451#discussion_r86619897
10. https://github.com/okta/okta.github.io/pull/451#discussion_r86620009
11. https://github.com/okta/okta.github.io/pull/451#discussion_r86620026
12. https://github.com/okta/okta.github.io/pull/451#discussion_r86620257

**Comment by mystiberry-okta** [comment]
Closing this PR as I hosed it right proper.

**Pull request closed without merging**